ontopy/utils.py

```diff
@@ -786,6 +786,14 @@ def directory_layout(onto):
 
         where `ontoA`, `ontoB` and `ontoC` are imported Ontology objects.
     """
+    all_imported = [
+        imported.base_iri for imported in onto.indirectly_imported_ontologies()
+    ]
+    # get protocol and domain of all imported ontologies
+    namespace_roots = set()
+    for iri in all_imported:
+        protocol, domain, *_ = urllib.parse.urlsplit(iri)
```

Comment:
Is the syntax `*_` valid in all versions of Python that we want to support?

Reply:
That is a typo. I'll remove the star.

Reply:
Nope, it was not a typo. Apparently this should work...

Reply:
I remember seeing this syntax being described in a release note for a new Python release. But I can't remember what release. What is the oldest version of Python we support?

Reply:
It passes in all the tests....